chibuezeayogu/React-Redux
# ch(webpackAndBabel): setup babel and webpack

**What does the PR do?**
- Adds `Webpack and Babel ` configuration

**Description of Task to be completed?**
- Ensure `Webpack and Babel ` is properly configured 

**How should this be manually tested?**
- Clone the repository [here](https://github.com/chibuezeayogu/React-Redux.git)
-  cd into `React-Redux` and checkout to this branch `ch-setup-webpack-and-babel-161040302`
- run the command `yarn dev` Webpack should compile successfully 

**What are the relevant pivotal tracker stories?**
- [#161040302](https://www.pivotaltracker.com/story/show/161040302)

**Screenshots**
- N/A
